docs/UserGuide.md

```diff
@@ -122,6 +122,23 @@ EduContacts is a **desktop app for educators in tertiary institutions to manage
 
 6. Refer to the [Features](#features) below for details of each command.
 
+--------------------------------------------------------------------------------------------------------------------
+
+## Summary of a `Person`
+
+This table will explain the fields that a `Person` in EduContacts possesses and its respective constraints. Each person
+in EduContacts is assumed to be Singaporean. All fields should not be blank upon the initialisation of a Student, except for `Module`.
+
+Field      | Details
+-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------
+**StudentID**   | The StudentID that belongs to the `Person`. The input for this field can only contain digits and should be exactly 8 digits long. <br><br> This field also serves as the unique identifier for a `Person`.
+**Name**   | The name that belongs to the `Person`. The input for this field can only contain alphanumeric characters.
+**Course** | The course that the `Person` studies. The input for this field can only contain alphabetical characters.
```

Comment:
```suggestion
**Name**   | The name that belongs to the `Person`. The input for this field can only contain alphanumeric characters and whitespaces.
**Course** | The course that the `Person` studies. The input for this field can only contain alphabetical characters and whitespaces.
```
I think we should specify that these two fields allow whitespaces to differentiate from the other fields that do not allow whitespaces e.g. module